streams/src/test/java/org/apache/kafka/streams/processor/internals/StateConsumerTest.java

```diff
@@ -69,6 +74,13 @@ public void shouldSeekToInitialOffsets() {
         assertEquals(30L, consumer.position(topicTwo));
     }
 
+    @Test(expected = IllegalArgumentException.class)
+    public void shouldSeekToInitialOffsetsWithMissingTopic() {
```

Comment:
nit: should we change the name of this test as it should not seek to initial offsets.  Do we need this test anyway as your updates to `initialize` prohibit this case?

Reply:
Good catch! I am going to delete this test. And I will add a unit test in `GlobalStreamThreadTest`